scripts/sandbox.ts

```diff
@@ -13,6 +13,7 @@ import {
 import prompts from 'prompts';
 import type { AbortController } from 'node-abort-controller';
 import command from 'execa';
+import dedent from 'ts-dedent';
```

Comment:
We should add https://www.npmjs.com/package/@ianvs/prettier-plugin-sort-imports so we don't need to worry about manually organizing imports.  :-D